README.md

```diff
@@ -1 +1,19 @@
-Invite URL: https://discord.com/api/oauth2/authorize?client_id=897382362894204998&permissions=380104723520&scope=bot
+# TikiTok Bot
+
+A Discord bot that watches for TikTok URLs and sends a message to the channel with the corresponding video attached
+
+Click  to invite the bot [![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.com/api/oauth2/authorize?client_id=897382362894204998&permissions=380104723520&scope=bot)
+
+## Environment Variables
+
+To run this project, you will need to add the following environment variables
+
+`DISCORD_TOKEN`
```

Comment:
Can you please change this to something more specific? DISCORD_TOKEN is slightly ambigious, and is just what we use internally for development purposes.

For people looking to host this bot themselves, they will need to create a discord app, then the discord token will be the bot's client secret attached to the app.